src/ocean/text/xml/PullParser.d

```diff
@@ -1,17 +1,13 @@
 /*******************************************************************************
 
-        Copyright:
-            Copyright (C) 2007 Aaron Craelius and Kris Bell
-            Some parts copyright (c) 2009-2016 Sociomantic Labs GmbH.
-            All rights reserved.
-
-        License:
-            Tango Dual License: 3-Clause BSD License / Academic Free License v3.0.
-            See LICENSE_TANGO.txt for details.
-
-        Version: Initial release: February 2008
-
-        Authors: Aaron, Kris
```

Comment:
I thought we'd deliberately left in the tango authors info?

Reply:
I don't remember details but I remember that SVN logs has shown that real authors are quite different from whatever is left in DDOC. And git history is still present anyway. @leandro-lucarella-sociomantic do you know how important is this part legal-wise?

Reply:
(restored just in case)